tests/external/iree-test-suites/onnx_ops/onnx_ops_cpu_llvm_sync.json

```diff
@@ -118,6 +118,14 @@
     "onnx/node/generated/test_nonmaxsuppression_two_batches",
     "onnx/node/generated/test_nonmaxsuppression_two_classes",
     "onnx/node/generated/test_nonzero_example",
+    "onnx/node/generated/test_pow_types_float32_int32",
+    "onnx/node/generated/test_pow_types_float32_int64",
+    "onnx/node/generated/test_pow_types_float32_uint32",
+    "onnx/node/generated/test_pow_types_float32_uint64",
+    "onnx/node/generated/test_pow_types_int32_float32",
+    "onnx/node/generated/test_pow_types_int32_int32",
```

Comment:
Regressions in simple ops like `pow` are concerning, especially for common type configurations like `i32`. Some type combinations went from runtime errors to compile errors, but a few tests were passing and are now failing.

https://github.com/iree-org/iree-test-suites/blob/main/onnx_ops/onnx/node/generated/test_pow_types_int32_int32/model.mlir
```mlir
module {
  func.func @test_pow_types_int32_int32(%arg0: !torch.vtensor<[3],si32>, %arg1: !torch.vtensor<[3],si32>) -> !torch.vtensor<[3],si32> attributes {torch.onnx_meta.ir_version = 8 : si64, torch.onnx_meta.opset_version = 17 : si64, torch.onnx_meta.producer_name = "backend-test", torch.onnx_meta.producer_version = ""} {
    %none = torch.constant.none
    %0 = torch.operator "onnx.Pow"(%arg0, %arg1) : (!torch.vtensor<[3],si32>, !torch.vtensor<[3],si32>) -> !torch.vtensor<[3],si32> 
    return %0 : !torch.vtensor<[3],si32>
  }
}
```

Can you link to the logs for the new compilation failures (or paste them if you ran locally)?

Reply:
I suspect https://github.com/llvm/torch-mlir/commit/ee08942c8fa51ae6fcdc0c231b138be16ec5a7ae is the culprit here.

Reply:
Given the justification lets just disable the test. We can rely on the actual onnx test suite instead of this weird one. I'll xfail appropriately.

Reply:
> We can rely on the actual onnx test suite instead of this weird one.

Can you clarify this? Rely on the ONNX model tests, not the op tests? I don't see how that unit test is "weird"... it's a pretty standard usage of the `Pow` op - a fundamental math operation.

I'm still not comfortable accepting a CPU regression that isn't present on GPU targets. We should at least have someone from CPU codegen take a look.